lib/testscript_engine/testscript_runnable.rb

```diff
@@ -170,6 +175,20 @@ def load_fixtures
     end
   end
 
+  def load_profiles
+    script.profile.each do |profile|
+      next warning(:no_static_profile_id) unless profile.id
+      next warning(:no_static_profile_reference) unless profile.reference
+
+      profile_server = FHIR::Client.new("")
```

Comment:
There's definitely a need for a client class, which should contain support for one-off requests like this. It feels somewhat wasteful to create new clients we don't keep track of for single requests. But, I think that exceeds the spirit of this issue and should be completed later.

Reply:
> There's definitely a need for a client class, which should contain support for one-off requests like this. It feels somewhat wasteful to create new clients we don't keep track of for single requests. But, I think that exceeds the spirit of this issue and should be completed later.

There are alternatives I can use: net, httparty, etc. It's a simple http request and I just used what's already there. Yes, I agree with that FHIR::Client is a bit heavy to run for a simple http.